ABOUT.md

```diff
@@ -27,8 +29,11 @@ Try out the most recent release with Boa's live demo
 [engine]: https://docs.rs/boa_engine/latest/boa_engine/index.html
 [gc]: https://docs.rs/boa_gc/latest/boa_gc/index.html
 [interner]: https://docs.rs/boa_interner/latest/boa_interner/index.html
+[interop]: https://docs.rs/boa_interop/latest/boa_interop/index.html
```

Comment:
The interop link here is still in the about files here.

Reply:
good spot!